scripts/dockermake.sh

```diff
@@ -0,0 +1,3 @@
+# Run `make` in a Docker container
+CWD="$(pwd)"
+docker run -u "$(id -u):$(id -g)" -v "$CWD/${DATA_DIR:-data}:/swedgene/data" -v "$CWD/Makefile:/swedgene/Makefile" swedgene-data make "$@"
```

Comment:
If you delete the data folder or are a new user without data folder you'll get an error like this: 
```
❯ ./scripts/dockermake.sh SPECIES=clupea_harengus build
mkdir: cannot create directory 'data/.downloads': Permission denied
scripts/make_download_targets.sh: line 32: data/targets.mk: Permission denied
tee: data/targets.mk: Permission denied
mkdir: cannot create directory 'data/.downloads': Permission denied
scripts/make_download_targets.sh: line 49: data/.downloads/clupea_harengus/GCF_900700415.2_Ch_v2.0.2_genomic.gff.gz: No such file or directory
Makefile:19: data/targets.mk: No such file or directory
make: *** [Makefile:120: data/targets.mk] Error 1
```

Adding in the line before the docker run command fixes it for me at least:

`mkdir -p "${CWD}/${DATA_DIR:-data}"`

Reply:
Thanks for the report and suggested fix, that I included verbatim. When the mount point on the host (in this case, `data`), Docker creates it; but since the Docker daemon runs as `root`, the created directory can only be written to by the `root` user. Since the container command is run with the host user privileges (through the `-u` flag), all attempts to create files in `data` failed. That is what I learned 🤓 !